pkg/network/tracer/offsetguess/offsetguess.go

```diff
@@ -123,6 +128,30 @@ func enableProbe(enabled map[probes.ProbeFuncName]struct{}, name probes.ProbeFun
 	enabled[name] = struct{}{}
 }
 
+func boolToUint64(in bool) uint64 {
```

Comment:
There is already a helper somewhere for this I think.

Reply:
the only thing I could find was `boolToBinary` in the usm package, but that returns `uint8`